README.md

```diff
@@ -19,6 +19,10 @@ Run ```node ./scripts/kick-off.js``` from project's rootpath to remove the exist
 Before running the app, make sure you must have a postgres db created. Then, set the `$NODE_API_DB_URL` environmental variable. It should look something like: `postgres://username:password@host:port/databasename`.
 For more information feel free to glance at the [`app/orm.js`](https://github.com/Wolox/express-js-bootstrap/blob/master/app/orm.js#L6) file.
 
+### Migrations
+
+To create a migration, run ```sequelize migration:create --name="my-migration-name"```. To run migrations, run ```npm run migrations```.
```

Comment:
Oh! Yes, the `migration-create` does actually that, it creates a migration file with the format `timestamp-my-migration-name`. If you create a migration file manually, you have to add the timestamp manually too. Besides, the script adds template migration code to the file